packages/census/scripts/modify-snarkjs-package.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
+const path = require('path');
+
+let packagePath = path.dirname(require.resolve("snarkjs"));
+
+while(!fs.existsSync(packagePath + '/package.json')) {
+    packagePath = packagePath.split('/');
```

Comment:
It's just a script, and not a critical code path... but I would use a different variable to assign an array... instead of the original string path one.